### PR TITLE
purge: ens 

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

purge ens feature

Tested locally

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -58,7 +58,7 @@ fix-stylelint: ## Run stylelint --fix against the project directory. Requires no
 	@npm run stylelint:fix
 
 fix-yapf: ## Run yapf against any included or newly introduced Python code.
-	@docker-compose exec web yapf -i -r -e "app/**/migrations/*.py" -e "app/app/settings.py" -p app/app/ app/avatar/ app/credits/ app/dataviz/ app/enssubdomain/ app/ethos/ app/github/
+	@docker-compose exec web yapf -i -r -e "app/**/migrations/*.py" -e "app/app/settings.py" -p app/app/ app/avatar/ app/credits/ app/dataviz/ app/github/
```
Comment (danlipert) [inline Makefile:61]: i dont even know what this is lol

File: app/faucet/templates/faucet_form.html [hunk 2]
```diff
@@ -88,14 +88,6 @@ <h3>{% trans "Faucet Request" %} <span id="alpha">{% trans "Alpha" %}</span></h3
             <div id="success_container" class="faucet_response text-center">
               <h5 class="mt-5 mb-4">{% trans "Request Received" %}</h5>
               {% include 'svgs/success.svg' %}
-              <div class="fail_message mt-4">
-                <p class="mb-0">{% trans "Your request for a faucet distribution has been received." %}</p>
-                <p class="mb-3">{% trans "The Gitcoin team will review your request shortly." %}</p>
-                <p class="mb-0">{% trans "While you wait, why not create an *free* Gitcoin ENS name?" %}</p>
```
Comment (danlipert) [inline app/faucet/templates/faucet_form.html:94]: Should we just remove the ENS portion of the message and leave the rest?

File: app/grants/management/commands/sent_cart_reminder.py [hunk 3]
```diff
@@ -21,14 +21,14 @@
...
-from django.db.models import Max, F
+from django.db.models import F, Max
 from django.utils import timezone
 
 from dashboard.utils import get_tx_status, has_tx_mined
 from grants.clr import predict_clr
-from grants.models import Contribution, Grant, CartActivity, Subscription
-from grants.views import clr_active, round_end, next_round_start
-from marketing.mails import warn_subscription_failed, remember_your_cart
+from grants.models import CartActivity, Contribution, Grant, Subscription
```
Comment (danlipert) [inline app/grants/management/commands/sent_cart_reminder.py:29]: booooo

File: Makefile (hunk 1 above)
Comment (thelostone-mc) [reply Makefile:61]: lol

Comment (thelostone-mc) [reply Makefile:61]: didn't want random errors pooping up so removed ir

File: app/grants/management/commands/sent_cart_reminder.py (hunk 3 above)
Comment (thelostone-mc) [reply app/grants/management/commands/sent_cart_reminder.py:29]: lol those were auto formatted by yapf :P

File: app/faucet/templates/faucet_form.html (hunk 2 above)
Comment (thelostone-mc) [reply app/faucet/templates/faucet_form.html:94]: addressed !

**Comment by thelostone-mc** [comment]
feedback comment addressed !

File: app/enssubdomain/management/commands/warn_when_wallet_out_of_eth.py [hunk 4]
```diff
@@ -1,42 +0,0 @@
-'''
```
Comment (octavioamu) [inline app/enssubdomain/management/commands/warn_when_wallet_out_of_eth.py:1]: not sure about this file removal, we don't use that for kudos mint and stuff?

File: scripts/crontab [hunk 5]
```diff
@@ -43,7 +43,6 @@ PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/us
 15 2 * * * cd gitcoin/coin; bash scripts/run_management_command.bash cleanup_stray_activities  >> /var/log/gitcoin/cleanup_stray_activities.log  2>&1
 15 4 * * * cd gitcoin/coin; bash scripts/run_management_command.bash cleanup_db_space  >> /var/log/gitcoin/cleanup_db_space.log  2>&1
 15 6 * * * cd gitcoin/coin; bash scripts/run_management_command.bash clearsessions  >> /var/log/gitcoin/clearsessions.log  2>&1
-16 * * * * cd gitcoin/coin; bash scripts/run_management_command.bash warn_when_wallet_out_of_eth  >> /var/log/gitcoin/warn_when_wallet_out_of_eth.log  2>&1
```
Comment (octavioamu) [inline scripts/crontab:46]: probably we need this job because also checks kudos wallet and grant wallet

**Pull request merged**